index.bs

```diff
@@ -72,9 +91,20 @@ of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
 
 A [=latest reading=] per [=sensor=] of orientation type includes one [=map/entries=]
 whose [=map/key=] is "quaternion" and whose [=map/value=] contains a four element [=list=].
-The elements of the [=list=] are equal to components of a unit quaternion representing orientation
-of the device in a three dimensional Cartesian coordinate system (x, y, z), where x-axis points east,
-y-axis points north and z-axis points up and is perpendicular to the plane made up of x and y axis.
+The elements of the [=list=] are equal to components of a unit quaternion [cos(θ/2), x**sin(θ/2), y*sin(θ/2), z*sin(θ/2)]
```

Comment:
maybe better `[cos(θ/2), Vx sin(θ/2), Vy sin(θ/2), Vz sin(θ/2)]`, where `V` is the unit vector representing the axis of rotation